pkgs/by-name/le/level-zero/setup-hook.sh

```diff
@@ -0,0 +1,7 @@
+if [ -d /run/opengl-driver/lib ]; then
+  export LD_LIBRARY_PATH="/run/opengl-driver/lib:$LD_LIBRARY_PATH"
+fi
+
+if [ -d /run/opengl-driver-32/lib ]; then
+  export LD_LIBRARY_PATH="/run/opengl-driver-32/lib:$LD_LIBRARY_PATH"
```

Comment:
```suggestion
  export LD_LIBRARY_PATH="/run/opengl-driver/lib${LD_LIBRARY_PATH:+:}${LD_LIBRARY_PATH:-}""
fi

if [ -d /run/opengl-driver-32/lib ]; then
  export LD_LIBRARY_PATH="/run/opengl-driver-32/lib${LD_LIBRARY_PATH:+:}${LD_LIBRARY_PATH:-}""
```
this way we make sure to never add an empty `:` at the end